clippy_lints/src/pub_underscore_fields.rs

```diff
@@ -20,13 +24,15 @@ declare_clippy_lint! {
     /// ```
     /// Use instead:
     /// ```rust
-    /// struct FileHandle_Foo {
+    /// struct FileHandle {
     ///     _descriptor: usize,
     /// }
-    /// 
+    /// ```
+    ///
     /// // OR
-    /// 
-    /// struct FileHandle_Bar {
+    ///
+    /// ```
```

Comment:
I'm not sure if the markdown page will get this, so just in case ミ๏ｖ๏彡

```suggestion
    /// ```rust
```